core-bundle/src/Resources/config/services.yml

```diff
@@ -493,6 +493,19 @@ services:
             - '%contao.prepend_locale%'
         public: true
 
+    Contao\CoreBundle\Search\Indexer\DefaultIndexer:
```

Comment:
```suggestion
    contao.search.default_indexer:
```

Reply:
No we agreed to use class names for new services.

Reply:
Fixed.